packages/libs/error-stack/examples/demo.rs

```diff
@@ -16,14 +16,33 @@ impl fmt::Display for ParseExperimentError {
 
 impl Context for ParseExperimentError {}
 
-fn parse_experiment(description: &str) -> Result<(u64, u64), ParseExperimentError> {
-    let value = description
-        .parse()
-        .into_report()
-        .attach_printable_lazy(|| format!("{description:?} could not be parsed as experiment"))
+fn parse_experiment(description: &str) -> Result<Vec<(u64, u64)>, ParseExperimentError> {
+    let values = description
+        .split(' ')
+        .map(|value| {
+            value
+                .parse::<u64>()
+                .into_report()
+                .attach_printable_lazy(|| format!("{value:?} could not be parsed as experiment"))
+        })
+        .map(|value| value.map(|ok| (ok, 2 * ok)))
+        .fold(Ok(vec![]), |acc, value| match (acc, value) {
```

Comment:
Would you mind renaming `acc` to `accum`? I always confuse `acc` with `account` 🙂

Reply:
fixed in [d1cf758](https://github.com/hashintel/hash/pull/1101/commits/d1cf758932175d95247ed717ca86bc19d0f628ac)